students/anbaopham/Lesson 3/Assignment_1.py

```diff
@@ -0,0 +1,124 @@
+import sys
+donor_db = [("William Gates, III", [653772.32, 12.17]),
+            ("Jeff Bezos", [877.33]),
+            ("Paul Allen", [663.23, 43.87, 1.32]),
+            ("Mark Zuckerberg", [1663.23, 4300.87, 10432.0]),]
+
+def get_name_list():
+    name_list =[]
+    for i in donor_db:
+        name_list.append(i[0])
+    return name_list
+
+
+def sort_key(donor_db):
+    return donor_db[0].split(" ")[1]
+
+
+prompt = "\n".join(("Welcome to the mailroom!",
+          "Please choose from below options:",
+          "1 - Send a Thank You",
+          "2 - Create a Report",
+          "3 - Exit",
+          ">>> "))
+
+def add_donation_amount(FullName, donation_amount):
```

Comment:
FullName should be snake_case: full_name

Reply:
fixed